networkx/generators/classic.py

```diff
@@ -490,3 +511,37 @@ def wheel_graph(n,create_using=None):
         G.add_edge(1,n-1)
     return G
 
+
+# TODO On Python 3, we can define this as
+#
+#     def complete_multipartite_graph(*block_sizes, create_using=None):
+#
+def complete_multipartite_graph(*block_sizes, **kw):
+    """Returns the complete multipartite graph with the specified block sizes.
```

Comment:
Sorry, one last thing. Can you clean up this docstring to make it more in line with how we typically have them in NetworkX. So a Parameters, Returns, and Examples section---the examples section will be helpful for people see esp when dealing with the `*block_sizes`.